bitbot/links.go

```diff
@@ -73,6 +73,7 @@ func shortenURL(uri string) string {
 	if err != nil {
 		b.Config.Logger.Warn("Coudln't shorten url", "error", err)
 	}
+	defer resp.Body.Close() //nolint:errcheck
```

Comment:
deferring this call without checking to see if `resp.Body` is non-nil can cause a segfault. Usually the error check exits the function.